### PR TITLE
Me: Add new section, "Get Apps"

### PR DESCRIPTION
**Pull request by jasmussen**
This PR aims to add a new subsection under the me tag, subheadline "Special", labelled "Get Apps". The purpose is to create one central place to reference desktop and mobile application clients for WordPress and WordPress.com.

Mockup:

mockup

Screenshot of current status:

screenshot

Before I proceed, I would like feedback on this PR as-is. Both on content, design, and code, thank you. Depending on feedback, at some point I'll need to add a few images to this section, SVG or PNG in the case of app store link buttons. What would be the best approach for adding these images?

### CODE REVIEW

File: client/me/apps/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Apps
```
Comment (mtias) [inline client/me/apps/README.md]: I'd call the component `get-apps` for a bit more clarity.

File: client/me/apps/style.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.apps .apps__title {
```
Comment (mtias) [inline client/me/apps/style.scss]: It shouldn't need the extra level of specificity, just `.apps__title` would be fine.

File: client/me/apps/style.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	text-align: center;
+	font-size: 24px;
+	font-family: Merriweather, Georgia, "Times New Roman", Times, serif;
```
Comment (mtias) [inline client/me/apps/style.scss]: There's a `$serif` var to output the font stack.

**Comment by jasmussen** [comment]
Addressed your feedback, Matias, superb as always thanks a bunch for doing this. Also rebased on master. Will start to add images now, and put them in `/public/images`.

File: assets/stylesheets/_components.scss [hunk 4]
```diff
@@ -121,6 +121,7 @@
 @import 'me/action-remove/style';
 @import 'me/application-password-item/style';
 @import 'me/application-passwords/style';
+@import 'me/get-apps/style';
```
Comment (mtias) [inline assets/stylesheets/_components.scss]: small nitpick, but we should now move it to forwards for the "g" letter. :)

File: client/me/get-apps/index.jsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+				<p className="get-apps__subheading">{ this.translate( 'Manage all your WordPress.com and Jetpack-enabled sites in one place. On your desktop, or on the go.' ) }</p>
+
+				<h2 className="form-section-heading">{ this.translate( 'Desktop apps' ) }</h2>
+				<Card className="get-apps__desktop">
+
+					<section className="get-apps__app">
+						<div className="get-apps__app-icon">
+						</div>
+						<p><strong>Mac</strong><br />
+						Requires 10.11 or newer</p>
```
Comment (mtias) [inline client/me/get-apps/index.jsx]: We should probably translate all these as well?

**Comment by mtias** [comment]
My pleasure! Looking good.

**Comment by jasmussen** [comment]
I think this is ready for final review. It looks like this now:

screen shot 2016-01-14 at 12 46 44

File: client/me/get-apps/index.jsx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+				<h1 className="get-apps__title">{ this.translate( 'Get Apps' ) }</h1>
+				<p className="get-apps__subheading">{ this.translate( 'Manage all your WordPress.com and Jetpack-enabled sites in one place. On your desktop, or on the go.' ) }</p>
+
+				<h2 className="form-section-heading">{ this.translate( 'Desktop apps' ) }</h2>
+				<Card className="get-apps__desktop">
+
+					<section className="get-apps__app">
+						<div className="get-apps__app-icon">
+							<img src="/calypso/images/me/get-apps-osx.svg" alt="" width="96" height="96" />
```
Comment (mjangda) [inline client/me/get-apps/index.jsx]: Should we fill in the `alt` attributes for the logos?

File: client/me/get-apps/index.jsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+				<h2 className="form-section-heading">{ this.translate( 'Mobile apps' ) }</h2>
+				<Card className="get-apps__mobile">
+
+					<section className="get-apps__app">
+						<div className="get-apps__app-icon">
+							<img src="/calypso/images/me/get-apps-ios.svg" alt="" width="96" height="96" />
+						</div>
+						<p><strong>{ this.translate( 'iOS' ) }</strong><br />
+						{ this.translate( 'Requires 9 or newer' ) }</p>
```
Comment (mjangda) [inline client/me/get-apps/index.jsx]: I wonder if we should drop the `requires` for the mobile apps. They could end up changing and we'd need to remember to update this page. Plus, the app store links will already include this info.

File: client/me/get-apps/index.jsx [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+							<img src="/calypso/images/me/get-apps-osx.svg" alt="" width="96" height="96" />
+						</div>
+						<p><strong>{ this.translate( 'Mac' ) }</strong><br />
+						{ this.translate( 'Requires 10.11 or newer' ) }</p>
+						<Button href="https://desktop.wordpress.com/d/osx">{ this.translate( 'Download' ) }</Button>
+					</section>
+
+					<section className="get-apps__app">
+						<div className="get-apps__app-icon">
+							<img src="/calypso/images/me/get-apps-windows.svg" alt="" width="96" height="96" />
```
Comment (mjangda) [inline client/me/get-apps/index.jsx]: It might be nice to linkify the the logos as well?

File: client/me/get-apps/index.jsx [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+				<h2 className="form-section-heading">{ this.translate( 'Mobile apps' ) }</h2>
+				<Card className="get-apps__mobile">
+
+					<section className="get-apps__app">
+						<div className="get-apps__app-icon">
+							<img src="/calypso/images/me/get-apps-ios.svg" alt="" width="96" height="96" />
+						</div>
+						<p><strong>{ this.translate( 'iOS' ) }</strong><br />
+						{ this.translate( 'Requires 9 or newer' ) }</p>
+						<a href="https://itunes.apple.com/us/app/wordpress/id335703880?mt=8"><img src="/calypso/images/me/get-apps-app-store.png" alt="" /></a>
```
Comment (mjangda) [inline client/me/get-apps/index.jsx]: We should definitely have `alt` for the App Store and Google Play logos since they are linked and don't have text.

**Comment by mjangda** [comment]
Looks pretty solid! Added some minor comments.

Also, noticed that on the smaller breakpoints the page title gets flushed a little too close to the nav bar:

get_apps_ _wordpress_com

**Comment by mjangda** [comment]
Oh, one more thing: for the Desktop download links, can we add `?ref=getapps` to the URL?

**Comment by jasmussen** [comment]
Added alt tags everywhere, added some topmargin on the mobile breakpoint, and added the ref to the download links. Did not linkify the logos. There are some relatively strict guidelines for how to use the app store badges, and in the case of Linux, we have two buttons, might even have three in the future.

**Comment by mjangda** [comment]
:+1:

**Pull request merged**